File: backend/src/main/java/ru/chsergeig/shoppy/dao/AccountRepository.java
package ru.chsergeig.shoppy.dao;

import org.jooq.Configuration;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Repository;
import ru.chsergeig.shoppy.jooq.tables.daos.AccountsDao;

@Repository
public class AccountRepository
        extends AccountsDao {

    public AccountRepository(@Autowired Configuration configuration) {
        super(configuration);
    }


}

File: backend/src/main/kotlin/ru/chsergeig/shoppy/dao/AccountRepository.kt
package ru.chsergeig.shoppy.dao

import org.jooq.DAO
import ru.chsergeig.shoppy.jooq.tables.pojos.Accounts
import ru.chsergeig.shoppy.jooq.tables.records.AccountsRecord

interface AccountRepository : DAO<AccountsRecord, Accounts, Int>
